### PR TITLE
1695538: Provide support for hypervisor_id option

### PR DESCRIPTION
**Pull request by pkliczewski**
We have 3 values supported for hypervisor_id but due to Kubernetes limitations
we can only support uuid and hostname. This PR adds support for both values.

### CODE REVIEW

**Comment by pkliczewski** [comment]
@wottop please review

**Comment by wottop** [comment]
needs rebasing

File: virtwho/virt/kubevirt/kubevirt.py [hunk 1]
```diff
@@ -84,14 +85,23 @@ def getHostGuestMapping(self):
             status = node['status']
             version = status['nodeInfo']['kubeletVersion']
             name = node['metadata']['name']
-            host_id = status['nodeInfo']['machineID']
-            address = status['addresses'][0]['address']
+
+            uuid = status['nodeInfo']['machineID']
```
Comment (wottop) [inline virtwho/virt/kubevirt/kubevirt.py:89]: This should be the system uuid. It need to be an immutable and unique identifier for the host system. If it is, then this is OK.

**Review by wottop — CHANGES_REQUESTED** [review]

File: virtwho/virt/kubevirt/kubevirt.py (hunk 1 above)
Comment (pkliczewski) [reply virtwho/virt/kubevirt/kubevirt.py:89]: Kubernetes guys suggest [1] to use `machineID` instead.

> SystemUUID reported by the node. For unique machine identification MachineID is preferred

[1] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#nodesysteminfo-v1-core

Comment (wottop) [reply virtwho/virt/kubevirt/kubevirt.py:89]: For this purpose, yes.

**Pull request merged**